.env.example

```
@@ -1,3 +1,4 @@
-DATABASE_URL=sqlite3://example.sqlite3
+DATABASE_URL=sqlite://example.sqlite3
 MAXMIND_USER_ID=12345
 MAXMIND_LICENCE_KEY=abcdef
+LINE_ENDING_TYPE=\n
```

This not really incorrect, but it would probably more accurate to change this to:
```suggestion
LINE_ENDING_TYPE=dos # or unix
```
As the ENV variable gets passed in and used here: https://github.com/rorymckinley/phisher_phinder/blob/master/lib/phisher_phinder/mail_parser.rb#L9